manifests/config.pp

```diff
@@ -6,14 +6,24 @@
 
   assert_private()
 
-  file { $::telegraf::config_file:
-    ensure  => file,
-    content => template('telegraf/telegraf.conf.erb'),
-    mode    => '0640',
-    owner   => 'telegraf',
-    group   => 'telegraf',
-    notify  => Class['::telegraf::service'],
-    require => Class['::telegraf::install'],
+  file {
```

Comment:
Ah sorry, I didn't even think about that when I was writing it. I'm happy to modify it so that it'll work on older Puppets.